Cargo.toml

```diff
@@ -164,6 +164,7 @@ task-compat = "0.1"
 cidr-utils = "0.4.2"
 pin-project = "0.4.26"
 k8s-openapi = { version = "0.9", features = ["v1_15"], optional = true }
+nats = { version = "0.7.3", optional = true }
```

Comment:
Nats 0.8 was released Sept 15th

Reply:
thanks, I bumped the version and ran `make check-all` before pushing